src/app/accounting/migrate-opening-balances/migrate-opening-balances.component.html

```diff
@@ -80,15 +80,15 @@
 
             <div fxFlexFill fxLayout="row wrap" fxLayoutGap="2%" fxLayout.lt-md="column" [formGroupName]="i">
 
-              <div fxFlex="8%">
+              <div fxFlex="8%" class="pt-1">
                 {{ (openingBalancesData.glAccounts[i].glAccountType.value !== openingBalancesData.glAccounts[i-1]?.glAccountType.value) ? openingBalancesData.glAccounts[i].glAccountType.value : '' }}
               </div>
 
-              <div fxFlex="18%">
+              <div fxFlex="18%" class="pt-1">
                 {{ openingBalancesData.glAccounts[i].glAccountCode }}
               </div>
 
-              <div fxFlex="18%">
+              <div fxFlex="18%" class="pt-1">
```

Comment:
![Screenshot from 2020-05-10 23-54-59](https://user-images.githubusercontent.com/54709463/81507266-be09cc00-9319-11ea-9640-3a467869d7ba.png)
use it like this, here it means margin top 20px, no need for scss decleration.

Reply:
refer docs for use case specific to this pr